cmd/connection.go

```diff
@@ -1,30 +1,24 @@
 package cmd
 
 import (
-	"fmt"
-	"os"
-
 	"github.com/qlik-oss/corectl/internal"
 	"github.com/qlik-oss/corectl/printer"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var setConnectionsCmd = &cobra.Command{
-	Use:   "set <path-to-connections-file.yml>",
-	Short: "Set or update the connections in the current app",
-	Long:  "Set or update the connections in the current app",
-	Example: `corectl connection set
-corectl connection set ./my-connections.yml`,
+	Use:     "set <path-to-connections-file.yml>",
+	Args:    cobra.ExactArgs(1),
+	Short:   "Set or update the connections in the current app",
+	Long:    "Set or update the connections in the current app",
+	Example: `corectl connection set ./my-connections.yml`,
 
 	Run: func(ccmd *cobra.Command, args []string) {
 		state := internal.PrepareEngineState(rootCtx, headers, true)
-		separateConnectionsFile := ""
-		if len(args) > 0 {
-			separateConnectionsFile = args[0]
-		}
+		separateConnectionsFile := args[0]
 		if separateConnectionsFile == "" {
-			separateConnectionsFile = getPathFlagFromConfigFile("connections")
+			internal.FatalError("Error: No connection file specified.")
```

Comment:
Do we need this check? Can't see the same check for e.g. entities

Reply:
Well, we probably should have it for entities. Without it things like:
```bash
corectl object set ""
```
can be run and return no error what so ever. In fact it prints "Saving app... Done" but it didn't do anything. (?)

This is also somewhat linked to #230 as we ask engine to look for something that can't possibly exist.

Reply:
I agree